psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -680,6 +683,12 @@ def scf_finalize_energy(self):
     core.print_out("\n\n")
     self.print_energies()
 
+    # reshape to be 2-d
```

Comment:
Confusing comment; I think you mean that the vector of length `N` is returned as a `N x 1` matrix?